sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -435,10 +433,7 @@ def _generate_pythonop(component_func, target_image, target_component_file=None)
           container=ContainerSpec(
               image=target_image,
               #command=['python3', program_file], #TODO: Include the command line
-              args=[{'value': input_name} for input_name in input_names],
-              file_outputs={ #TODO: Use proper output arguments (e.g. "{output: output_name}" ) instead of this workaround. Our 1st-party components should not be using the file_outputs workaround.
-                output_name: output_file,
-              }
+              args=[{'value': input_name} for input_name in input_names] + [{'output': output_name}],
```

Comment:
Is the spec under constant change? To review the codes, I need the format spec for the component artifact.

Reply:
No. The spec did not change. Even before you've implemented the python container op, I've talked to you extensively asking for your generated code to accept both input and output paths. It's the same way 99% of command-line apps work: `./program --input path/input.txt --param=11 --output path/output.txt`. Instead of this common standard, your code generator produced programs that can only write to `/output.txt`. You said that it was a bit hard for you to implement this and I agreed to let you hardcode '/output.txt' path, saying that we should make it configurable as soon as we have time. Here I'm following up on that.

Reply:
I remember demonstrating you the command line I wanted to be supported by the components produced by your codegen.
Instead of
`add.py 3 5 #implicitly writes to /output.txt. The path cannot be changed`
I asked for the program to support
`add.py 3 5 /outputs/sum.txt`

Reply:
Fair enough